### PR TITLE
Merge all omf list functions

### PR DESCRIPTION
**Pull request by bpinto**

### CODE REVIEW

**Comment by derekstavis** [comment]
Wooooow!! Such a nice PR!! :tada:

File: pkg/omf/cli/omf.packages.list.fish [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  test "$type" = "--plugin"; or for package in (basename {$OMF_CONFIG,$OMF_PATH}/themes/*)
+    set list $list $package
+  end
+
+  __omf.packages.sort $list
+end
+
+function omf.packages.list -a option type
+  switch $option
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.list.fish]:
```fish
switch "$option"
```

File: pkg/omf/cli/omf.packages.list.fish [hunk 2]
```diff
@@ -0,0 +1,72 @@
+function __omf.packages.sort
+  for package in (echo $argv | tr ' ' '\n' | sort); echo $package; end
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.list.fish]:
```fish
for package in (printf "%s\n" $argv | sort); echo $package; end
```

**Comment by bobthecow** [comment]
:+1:

File: pkg/omf/omf.fish [hunk 3]
```diff
@@ -118,7 +118,7 @@ function omf -d "Oh My Fish"
         set -l regex "[[:<:]]($theme)[[:>:]]"
         test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
 
-        omf.list_themes | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
+        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
         omf::off
       else if test (count $argv) -eq 2
         if not contains -- $argv[2] (omf.list_installed_themes)
```
Comment (derekstavis) [inline pkg/omf/omf.fish]:
```fish
if not contains -- $argv[2] (omf.packages.list --installed)
```

Comment (bpinto) [inline pkg/omf/omf.fish]: Nice catch!

File: pkg/omf/cli/omf.packages.list.fish (hunk 2 above)
Comment (bpinto) [inline pkg/omf/cli/omf.packages.list.fish]: Keeping the code as it is, both are ugly. :P

**Comment by derekstavis** [comment]
:+1:

**Pull request merged**